src/core/core.config.js

```diff
@@ -35,7 +35,7 @@ export function determineAxis(id, scaleOptions) {
   if (id === 'x' || id === 'y') {
     return id;
   }
-  return scaleOptions.axis || axisFromPosition(scaleOptions.position) || id.charAt(0).toLowerCase();
+  return scaleOptions.axis || axisFromPosition(scaleOptions.position) || id;
```

Comment:
Seems like the last or would not be necasarry since `foo` is not a valid axis type

```suggestion
  return scaleOptions.axis || axisFromPosition(scaleOptions.position);
```